### DISCUSSION TITLE
grbl-Mega-5x with Candle 2.0?

### DISCUSSION THREAD

**Discussion by jdeline** [fra589/grbl-Mega-5X#228] [Q&A] [answered]
I have version 1.2d of grbl-Mega-5x.  I can connect to the board to Candle 2.0 (under Windows 10), but cannot seem to do anything after that.  Are there compatibility issues I am unaware of?  Is there a config setting I can set to get it to work with Candle?

**Comment by jdeline** [comment]
I got a response on the Candle github page.  Candle checks to see that the grbl version is 1.1

**Comment by fra589** [comment, marked as answer]
Hu @jdeline, 

I just answered to your [Candle's issue](https://github.com/Schildkroet/Candle2/issues/20#issuecomment-968266990)...
  
@++;  
Gauthier.
